rl_loop.py

```diff
@@ -33,6 +34,8 @@
 SGF_DIR = os.path.join(BASE_DIR, 'sgf')
 TRAINING_CHUNK_DIR = os.path.join(BASE_DIR, 'data', 'training_chunks')
 
+GAMES_BEFORE_TRAINING = 4000
```

Comment:
I'm big on comments, of course. Can we add one here?

Reply:
Done!